app.rb

```diff
@@ -0,0 +1,158 @@
+require './model/book'
+require './model/game'
+require './model/music_album'
+require './model/genre'
+require './model/author'
+require './model/label'
+require './data'
+
+class App
+  def initialize
+    @data = Data.new
+    @data.load_authors
+    @data.load_games
+    @data.load_albums
+    @data.load_genres
+    @data.load_books
+    @data.load_labels
+  end
+
+  def inputs(show)
+    outputs = []
+    show.each do |s|
+      print s
+      outputs.push(gets.chomp)
+    end
+    outputs
+  end
+
+  def add_author
+    puts 'Enter first name'
+    first_name = gets.chomp
+    puts 'Enter last name'
+    last_name = gets.chomp
+    new_author = Author.new(nil, first_name, last_name)
+    @data.add_author(new_author)
+    @data.authors << new_author
+    new_author
+  end
+
+  def add_game
```

Comment:
- [ ] Please kindly fix the console error below. I can't add a game.


![Screenshot from 2022-11-24 23-24-52](https://user-images.githubusercontent.com/63087491/203870279-dd440a9c-0acb-4d96-9b60-153f8f01bea5.png)

Reply:
## Required
- [ ] Address this I am still getting the same error
<img width="1093" alt="Screen Shot 2022-11-25 at 11 29 03" src="https://user-images.githubusercontent.com/44745855/203947375-890513a9-dc69-4b06-b80b-ad4c42099ddc.png">